libs/search-widget/src/core/utils.ts

```diff
@@ -385,3 +385,13 @@ export function hasNotEnoughData(text: string): boolean {
   text = text.replace(/<br>/g, '').replace(/\n/g, '');
   return text === 'Not enough data to answer this.';
 }
+
+export function downloadAsJSON(data: any) {
+  const element = document.createElement('a');
+  element.setAttribute('href', 'data:application/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(data)));
+  element.setAttribute('download', `dump-${new Date().toISOString()}.json`);
+  element.style.display = 'none';
+  document.body.appendChild(element);
```

Comment:
I don't think you need to add the element in the body to be able to click on it.